lib/pbench/test/unit/server/test_requests.py

```diff
@@ -91,7 +94,7 @@ def test_bad_extension_upload(client, bad_extension):
 
     @staticmethod
     def test_empty_upload(client, pytestconfig):
-        expected_message = "Payload Content-Length empty, nothing was uploaded"
+        expected_message = "Upload failed, Content-Length received in header is 0"
```

Comment:
It'd probably be nice to have test cases for Content-Length missing, and > MAX as well.

We should also have cases for the uploaded file MD5 not matching, and for a file length != Content-Length.

This could probably be done later, but it'd be better to go in with validation that it all works.

Reply:
indeed, I am looking into how we can add a custom `Content_Length` header value in flask test client, the request automatically overrides anything we send as value for `Content-Length`. Thats why even in `test_empty_upload` I had to create an empty `tar.xz` file in flask tmp directory and use that to upload. I coundn't simply pass `Content-Length: 0` in header of the request

Reply:
Can you file an issue to track adding this kind of testing later?

Reply:
Created an issue to track https://github.com/distributed-system-analysis/pbench/issues/1851